src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Payum.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
+>
+    <resource class="Sylius\Bundle\ApiBundle\Command\Payum\PayumRequest" shortName="Payum">
+        <attribute name="messenger">input</attribute>
+        <attribute name="validation_groups">sylius</attribute>
+        <attribute name="read">false</attribute>
+
+        <collectionOperations />
+
+        <itemOperations>
+
+            <itemOperation name="shop_capture">
+                <attribute name="method">GET</attribute>
```

Comment:
`capture` within Payum is "normally" designed to start a payment process, but some gateways are using it also to update the payment status even though Payum tokens are not meant to be used like that.
Anyway, since the capture is used in these two cases: starting or finishing a payment process then a `PATCH` is better.

Reply:
Hm, I thought you updated the method already. :)